### DISCUSSION TITLE
How to add sum row in each page

### DISCUSSION THREAD

**Discussion by VladaMitic** [dolanmiu/docx#1355] [Q&A]
Hi, First thanks for the great package.
I am trying to render a multi-page table in word, and I really didn't have a problem with that. But I need to add the last row on each page that is the sum of certain columns on that page, accept the last page where that row is a sum of all certain columns. Is there any solution or recommendation for this?

**Comment by dolanmiu** [comment]
docx don't have any support for this, so it seems you may have to write custom logic to calculate the columns etc
